env0/resource_project_policy.go

```diff
@@ -152,10 +158,18 @@ func resourcePolicyUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 	}
 
 	// Validate that default ttl is "less than or equal" max ttl.
-	defaultTtlIndex := getTtlIndex(&payload.DefaultTtl)
-	maxTtlIndex := getTtlIndex(&payload.MaxTtl)
-	if maxTtlIndex < defaultTtlIndex {
-		return diag.Errorf("default ttl must not be larger than max ttl")
+	defaultTtl, err := getPolicyTtl(payload.DefaultTtl)
+	if err != nil {
+		return diag.Errorf("invalid default ttl: %v", err)
+	}
+
+	maxTtl, err := getPolicyTtl(payload.MaxTtl)
+	if err != nil {
+		return diag.Errorf("invalid max ttl: %v", err)
+	}
+
+	if maxTtl < defaultTtl {
+		return diag.Errorf("default ttl must not be larger than max ttl: %d %d", defaultTtl, maxTtl)
```

Comment:
anyway to avoid the code duplication here and in organization?

Reply:
I think there is. I'll check...

Reply:
Refactored and removed code duplication.
d12f08e06636e169d9c306a715ceb5f687e37d60